corn/styles.css

```diff
@@ -1,70 +1,134 @@
-:root{
-    
-    --font-family:'Inter', sans-serif;
-    --subtittleLi-subtittle-color:#575767;
-    --background-color:#F8F8F8;
-    --hr-color:#D0D0D0;
-    --color-subtitle-li:var(--subtittleLi-subtittle-color);
-    --font-color-h1:#0E0E11;
-    --subtittle-color:var(--subtittleLi-subtittle-color);
-    --font-size-html:18px;
-    --font-size-p: 14px;
-    --font-size-h1:32px;
-    --font-weight-h1-h2:700;
-    --font-weight-p:600;
-    --font-weight-li:500;
-    --color-p-label:#B9B9BE;
-    
-    
+:root {
```

Comment:
Looks like it positioned wrongly:

![Screenshot 2021-06-28 at 16 42 21](https://user-images.githubusercontent.com/84386984/123646887-4ace5c80-d830-11eb-97be-2df894dfcc9e.png)